### DISCUSSION TITLE
Dualshock3 connects as controller 4 via bluetooth

### DISCUSSION THREAD

**Discussion by prkmk** [nefarius/DsHidMini#322] [Q&A]
As the title suggests, whenever I connect my dualshock3 to PC wirelessly (dshidmini + bthps3) it logs as the controller 4 (the light indicates this). However, if I plug it in with a USB cable it's controller 1. This affects control mapping. How do I overcome this issue? I'm clueless how to even google this problem.

I tried resetting the controller (not sure if it worked), re-enabling the drivers and rebooting my PC but nothing changed.

**Comment by nefarius** [comment]
Please, **please** could you folks for just once read the bloody documentation, is that too much to ask, I'm changing profession to gardener or plumber, I'm done with software 😪

**Comment by prkmk** [comment]
Okay sorry I didn't read it properly. I thought it's a bug and tried to
find info about that, not that the LEDs are used to indicate the battery
charge. Nonetheless, the issue remains. The controller doesn't seem to work
properly via Bluetooth. Could you kindly point me to the answer, please? I
failed to find any helpful info from documentation and discussion of this
and bthps3.

**Reply by nefarius** [reply]
It's literally in the README and on the docs page, clear as day. Sorry but if you need me as a search engine, that is not my purpose on this platform. Good luck.

**Reply by nefarius** [reply]
See FAQ articles and the modes explanation. The driver works, it's up to you to configure it properly.
